doc/src/ROCMHeaderWrapper.rst

```diff
@@ -20,7 +20,9 @@ Commands
 
 Create a C/C++ wrapper file for each specified header file. The wrapper is simply a C/C++ header
 that emits a deprecation warning before including its corresponding header. The warning can be
-turned off by defining ``ROCM_NO_WRAPPER_HEADER_WARNING``.
+turned off by defining ``ROCM_NO_WRAPPER_HEADER_WARNING`` when using the wrapper header files.
+The wrapper may be configured to emit an error instead by defining ``ROCM_HEADER_WRAPPER_WARN_AS_ERROR``
+at the time of building the library which contains the header files to be wrapped.
```

Comment:
The compile-time and cmake configure-time options need to be clearly distinguished.

Reply:
I'd just like to clarify: distinguished in documentation only, or should they also have separate names? e.g. `ROCM_HEADER_WRAPPER_WERROR_CONFIG` and `ROCM_HEADER_WRAPPER_WERROR`?

Reply:
Distinguished in documentation. One sets the default for the other, so I think it's fine to give them the same name.